platform-hub-web/src/app/projects/projects-list.component.js

```diff
@@ -23,18 +26,66 @@ function ProjectsListController(roleCheckerService, Projects) {
   function loadProjects() {
     ctrl.loading = true;
 
-    Projects
-      .refresh()
-      .finally(() => {
-        ctrl.loading = false;
+    return Projects.getAll()
+    .then(projects => {
+      return Me.refresh().then(() => {
+        const currentUserId = Me.data.id;
+        return Promise.all(projects.map(projectIdto => {
+          return Projects.getMemberships(projectIdto.id);
+        })).then(memberships => {
+          ctrl.memberships = memberships;
+
+          ctrl.myProjects = getMyProjects(projects, currentUserId, memberships);
```

Comment:
here i'd probably have one functions `getProjects` and pass a flag like `myProjects: boolean`, that way can reuse most or whats in `getMy/NotMyProjects`

Reply:
Resolved- created one function `getProjects`